packages/augur-ui/src/modules/common/form.tsx

```diff
@@ -1565,7 +1567,9 @@ class IndividualTimeSelector extends React.Component<
   decrement = () => {
     const value = this.state.value;
     if (!this.props.hasOptions) {
-      const newValue = parseFloat(value) - 1;
+      let newValue = parseFloat(value) - 1;
+      if (newValue > this.props.max) newValue = this.props.min;
+      if (newValue < this.props.min) newValue = this.props.max;
```

Comment:
If the above comment makes sense, the values assigned here would also need to change.

Reply:
need to give the user ability to loop through the values and not get stuck.